libs/core/tests/unit_tests/_api/test_deprecation.py

```diff
@@ -280,14 +280,15 @@ def deprecated_method(self) -> str:
         assert len(warning_list) == 2
         warning = warning_list[0].message
         assert str(warning) == (
-            "The class `tests.unit_tests._api.test_deprecation.DeprecatedClass` was "
+            "The class `test_whole_class_deprecation.<locals>.DeprecatedClass` was "
```

Comment:
<locals>

Reply:
Is this a change request? This is how it renders for me and in CI. Does it act differently on your machine?